README.md

```diff
@@ -91,3 +91,10 @@ import HdDynamicForm from 'hd-blocks/components/form/HdDynamicForm.vue';
 
 ## Customize configuration
 See [Configuration Reference](https://cli.vuejs.org/config/).
+
+## Services
```

Comment:
I would add the link to the services/README here instead of listing all the services here. I would list the services as an index in the beginning of services/README file.

Reply:
Agreed, done :+1: